plugin/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go

```diff
@@ -37,7 +37,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
   "predicates": [
     {"name": "MatchNodeSelector"},
     {"name": "PodFitsResources"},
-    {"name": "PodFitsPorts"},
+    {"name": "PodFitsHostPorts"},
```

Comment:
@liggitt 

Good point, I will handle the backwards compatibility issue in the next update.